buildSrc/subprojects/plugins/src/main/kotlin/gradlebuild/security.gradle.kts

```diff
@@ -20,16 +20,21 @@ import java.util.concurrent.locks.ReentrantLock
 import kotlin.concurrent.withLock
 
 val lock = ReentrantLock()
+val stackWalker = StackWalker.getInstance()!!
```

Comment:
❌ I don't think this will work, since we always compile `buildSrc` with the same JDK Gradle is running on.

Moreover, the call to `StackWalker.getInstance()` should also be guarded by a `isJava9Compatible` call.

Reply:
mmm, ok I thought you said we would fork compilation for `buildSrc` too. This is annoying...

Reply:
the `StackWalker` init can just be avoided by doing `val stackWalker by lazy { ... }` but I decided to revert this change. We'll live with the performance regression, I don't have time to deal with the complexity of making this work on both Java 8 and Java 9+.